OneCommunityGlobal/HighestGoodNetworkApp
# Fix: Faye MonthlyEffort ESLint and Prettier issues

# Description
Lint fix for the MonthlyEffort Component

## Main changes explained:
Removed MonthlyEffort from the .eslintignore and .prettierignore files to enforce code quality and formatting checks.
Refactored MonthlyEffort from a class component to a functional component for simplicity and improved readability.
Removed unnecessary state and lifecycle methods that were no longer in use.
Added ESLint rule exceptions in index.js to allow specific export practices that align with the team's conventions.

## How to test:
Check out the current branch.
Run npm install to ensure dependencies are up-to-date.
Run the application locally and verify if the MonthlyEffort component functions as expected.
Review the MonthlyEffort component for any warnings or errors in the console.

## Note:
Include the information the reviewers need to know.
